releng/org.eclipse.xtext.target/org.eclipse.xtext.target-luna.target

```diff
@@ -92,8 +92,20 @@
 	</location>
 	
 	<location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-		<repository location="http://download.eclipse.org/modeling/tmf/xtext/updates/orbit/"/>
 		<unit id="org.junit" version="0.0.0"/>
+		<unit id="com.google.inject" version="0.0.0"/>
+		<unit id="com.google.guava" version="21.0.0.v20170206-1425"/>
+		<unit id="org.apache.commons.cli" version="0.0.0"/>
+		<unit id="org.objectweb.asm" version="6.0.0.v20180116-1719"/>
+		<unit id="org.objectweb.asm.tree" version="6.0.0.v20180116-1719"/>
+		<unit id="org.apiguardian" version="0.0.0"/>
+		<unit id="org.junit.jupiter.api" version="0.0.0"/>
+		<unit id="org.junit.jupiter.engine" version="0.0.0"/>
+		<unit id="org.junit.jupiter.params" version="0.0.0"/>
+		<unit id="org.junit.platform.commons" version="0.0.0"/>
+		<unit id="org.junit.platform.engine" version="0.0.0"/>
+		<unit id="org.opentest4j" version="0.0.0"/>
```

Comment:
what is this? 'opentest4j' ?

Reply:
https://github.com/ota4j-team/opentest4j
Runtime dependency by junit5, see https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-dependencies

Reply:
Ok